src/components/StyledText.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { FC, HTMLAttributes } from "react";
+import { css, StyleSheet } from "aphrodite";
+import { colors, typography } from "styles";
+
+interface Props {
+  error?: boolean;
+  style?: HTMLAttributes<HTMLParagraphElement>;
+}
+
+const StyledText: FC<Props> = ({ children, error, style }) => {
+  return (
+    <p
+      className={css(
+        styles.text,
+        style,
+        typography.body1,
+        error && styles.errorText
+      )}
+    >
+      {children}
+    </p>
+  );
+};
+
+const styles = StyleSheet.create({
+  text: {
+    fontFamily: "Roboto",
+    textTransform: "none",
+  },
+  errorText: {
+    marginTop: 3,
+    fontFamily: "Ubuntu",
```

Comment:
same

Reply:
In this case we don't have className={} for error text, so I couldn't attach typography.example like in previous. But I will be trying to do something with this.